v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -12,18 +12,17 @@ import {
   defaultSelectedStrokeWidth, defaultStrokeOpacity, defaultStrokeWidth
 } from "../../../utilities/color-utils"
 import {IDataConfigurationModel} from "../models/data-configuration-model"
+import {measureText} from "../../../hooks/use-measure-text"
 
 /**
  * Utility routines having to do with graph entities
  */
 
 export const maxWidthOfStringsD3 = (strings: Iterable<string>) => {
-  const text = selection().append('svg').append('text').style('font', '12px sans-serif')
   let maxWidth = 0
   for (const aString of strings) {
-    maxWidth = Math.max(maxWidth, text.text(aString).node()?.getComputedTextLength() ?? 0)
+    maxWidth = Math.max(maxWidth, measureText(aString, '12px sans-serif'))
```

Comment:
This font string occurs multiple times -- seems like a named constant would be a good idea, e.g. `kGraphD3Font` or something like that, so that if we ever need to change it there's one place to do so.

Reply:
Agreed